src/app/apps/rup/mapa-camas/sidebar/egreso/egresar-paciente.component.ts

```diff
@@ -214,26 +218,27 @@ export class EgresarPacienteComponent implements OnInit, OnDestroy {
                     if (this.subscription2) {
                         this.subscription2.unsubscribe();
                     }
-                    let fechaABuscar = moment(this.informeIngreso.fechaIngreso).add(1, 's');
+                    const fechaABuscarMin = moment(this.informeIngreso.fechaIngreso).add(-1, 's');
+                    let fechaABuscarMax = moment();
                     if (this.hayEgreso) {
-                        fechaABuscar = moment(this.registro.valor.InformeEgreso.fechaEgreso).add(-10, 's');
+                        fechaABuscarMax = moment(this.registro.valor.InformeEgreso.fechaEgreso).add(-10, 's');
```

Comment:
Todo es en una linea queda mas lindo :) 
```suggestion
                       const fechaABuscarMax = this.hayEgreso?moment(this.registro.valor.InformeEgreso.fechaEgreso).add(-10, 's'): moment();
```